.github/renovate-config.json5

```diff
@@ -16,7 +16,7 @@
   dependencyDashboard: false,
   enabledManagers: ["bun", "custom.regex", "github-actions", "npm"],
   forkProcessing: "enabled",
-  globalExtends: ["config:best-practices"],
+  globalExtends: [":pinDependencies", "config:best-practices"],
```

Comment:
it's not clear from the docs if we really need the : in the front? 🤔

Reply:
[It's part of the name](https://docs.renovatebot.com/presets-default/#pindependencies). What's making you think otherwise?

Check the [logs for the run on this PR](https://github.com/grafana/shared-workflows/actions/runs/12051300936/job/33602021745?pr=568) too - it tells you what it would have done.

Reply:
yeah couldn't understand if we should have it there or not, but since it's working I guess that's the right way to go 😁 thought it was a c/p issue!

Reply:
heh. thanks for being diligent!